Sun Jan  2 13:21:29 CET 2005  Norbert@Nemec-online.de
  * create link to mkhowto
Index: numarray/Doc/manual/Makefile
===================================================================
--- numarray.orig/Doc/manual/Makefile	2005-04-28 17:58:23.000000000 +0200
+++ numarray/Doc/manual/Makefile	2005-11-02 11:28:25.000000000 +0100
@@ -5,7 +5,7 @@
 
 # some programs, might need adjustment
 GZIP		= gzip -9f
-MKHOWTO		= python2 ./mkhowto
+MKHOWTO		= ./mkhowto
 
 
 # some information for generated html
@@ -67,24 +67,21 @@
 
 # some real targets
 
-mkhowto:
-		echo "You need to create a link to mkhowto,"
-		echo "see the README for details."
-
-numarray:	Makefile mkhowto $(SOURCES)
+.PHONY: numarray
+numarray:	Makefile $(SOURCES)
 		$(MKHOWTO) --html $(OPTIONS) numarray.tex
 
-numarray.dvi:	Makefile mkhowto $(SOURCES)
+numarray.dvi:	Makefile $(SOURCES)
 		$(MKHOWTO) --dvi $(OPTIONS) numarray.tex
 
-numarray.pdf:	Makefile mkhowto $(SOURCES)
+numarray.pdf:	Makefile $(SOURCES)
 		$(MKHOWTO) --pdf $(OPTIONS) numarray.tex
 
-numarray.ps.gz:	Makefile mkhowto $(SOURCES)
+numarray.ps.gz:	Makefile $(SOURCES)
 		$(MKHOWTO) --ps $(OPTIONS) numarray.tex
 		$(GZIP) numarray.ps
 
-numarray.txt.gz: Makefile mkhowto $(SOURCES)
+numarray.txt.gz: Makefile $(SOURCES)
 		$(MKHOWTO) --text $(OPTIONS) numarray.tex
 		$(GZIP) numarray.txt
 
Index: numarray/Doc/manual/mklinks
===================================================================
--- numarray.orig/Doc/manual/mklinks	2002-03-22 05:35:47.000000000 +0100
+++ numarray/Doc/manual/mklinks	2005-11-02 11:28:25.000000000 +0100
@@ -1,8 +1,8 @@
 #! /bin/sh
 rm -f checkargs.pm html2texi.pl icons mkhowto mkinfo
 
-ln -s $1/dist/src/Doc/html/icons icons
-ln -s $1/dist/src/Doc/tools/checkargs.pm checkargs.pm
-ln -s $1/dist/src/Doc/tools/html2texi.pl html2texi.pl
-ln -s $1/dist/src/Doc/tools/mkhowto mkhowto
-ln -s $1/dist/src/Doc/tools/mkinfo mkinfo
+ln -s $1/Doc/html/icons icons
+ln -s $1/Doc/tools/checkargs.pm checkargs.pm
+ln -s $1/Doc/tools/html2texi.pl html2texi.pl
+ln -s $1/Doc/tools/mkhowto mkhowto
+ln -s $1/Doc/tools/mkinfo mkinfo
